index.html

```diff
@@ -189,9 +186,37 @@ <h2 class=" connect">LET'S CONNECT</h2>
                 <a href="#" class="dropdown-btn"><div>Skills</div><img src="images/arrow.png"></a>
             </li>
           </ul>
-       
+        </div>
     </section>
-    
+    <section class="contact-me">
+    
+        <div class="form-header">
+        <h2 class="form-heading"> Contact me </h2>
+ 
+        <p class="pg-contact">If you have an application you are interested in developing, a feature that you
+          need built or a project that needs coding. 
+         I’d love to help with it!
+         </p>
+         </div>
+         <form action="https://formspree.io/f/mqkvkgdo" method="POST" class="contact-me-form">
+         <ul class="form-info">
+         <li>     
+            <!-- <label for="name" value="name">Name:</label> -->
+             <input type="text" id="name" value="Lorem Ipsum" name="name" required placeholder="enter your name" maxlength="30"/></li>
+         <li>
+             <!-- <label for="e-mail" value="e-mail">E-mail:</label> -->
+                 <input type="email" id="e-mail" value="loremipsum@gmail.com" name="email" required placeholder="enter your email"/> </li>
+          <li>
+                 <!-- <label for="message" value="message" >Message</label><br> -->
```

Comment:
- [ ] Kindly consider removing the `value` attribute from your input field as it is unnecessary in this case. Including the `value` attribute with a default value requires users to manually delete the pre-filled value before entering their own information, which can create a poor user experience. By removing the `value` attribute, you allow users to input their information directly without the need for extra steps, improving the overall usability of the form.
 
 - [ ] Also, Kindly consider removing all `commented-out` code from your `codebase` as it can make the `codebase` appear cluttered and unprofessional. `Commented` code serves no purpose in the actual execution of the program and can create confusion or unnecessary distractions for other developers. By removing `commented-out` code, you improve the readability and maintainability of the `codebase`, making it easier for future developers to understand and work with the code.